### PR TITLE
Create : 1768-merge-strings-alternately.java

### PR DESCRIPTION
**Pull request by Aakriti-malla**
Solution to Leetcode problem: 1768

- **File(s) Modified**: _1768-merge-strings-alternately.java_
- **Language(s) Used**: _java_
- **Submission URL**: _https://leetcode.com/problems/merge-strings-alternately/submissions/998999168/_

### CODE REVIEW

File: java/1768-merge-strings-alternately.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Solution {
+    public String mergeAlternately(String word1, String word2) {
+        int i = 0, j = 0;
+        StringBuilder res = new StringBuilder();
+
+        while (i < word1.length() && j < word2.length()) {
```
Comment (aakhtar3) [inline java/1768-merge-strings-alternately.java:6]: Suggestion: use || and check boundaries of your pointers to avoid substring at end of your function.

**Review by aakhtar3 — CHANGES_REQUESTED** [review]

File: java/1768-merge-strings-alternately.java (hunk 1 above)
Comment (aakhtar3) [reply java/1768-merge-strings-alternately.java:6]: And time and space complexity

Comment (aakhtar3) [reply java/1768-merge-strings-alternately.java:6]: You can also solve with a single index pointer

Comment (Aakriti-malla) [reply java/1768-merge-strings-alternately.java]: Addressed all the Suggestions and added the time and space complexity at the end.

**Review by aakhtar3 — APPROVED** [review]

**Pull request merged**